jfsd/appliedForces.py

```diff
@@ -206,7 +209,8 @@ def compute_hs_forces(
 
         #compute forces for each pair
         # relax constant (If lubrication is not on, k should to be ~ o(1) )
-        k = 1000.
+        k = jnp.where(HIs_flag > 0, 1000. , 100)
```

Comment:
You might want to explain this / write a comment here why this is the case.

Reply:
Added a more extensive comment [https://github.com/torrewk/Python-Jax-Fast-Stokesian-Dynamics/pull/13/commits/581bca0f2aa873674b5fa30f0f7d308d711c5fe1](url)